.github/workflows/ci_nightly.yml

```diff
@@ -0,0 +1,77 @@
+# This CI workflow is triggered by:
+#   - scheduled run
+#
+# In the scheduled run, we run all targets from amdgpu_family_matrix.py and amdgpu_family_matrix_xfail.py
+# As some of these builds are xfail, we allow errors to occur with `continue-on-error`, where the job will fail but the workflow is green
+
+name: CI Nigthly
+
+on:
+  # For AMD GPU families that expect_failure, we run builds and tests from this scheduled trigger
+  schedule:
+    - cron: "0 2 * * *" # Runs nightly at 2 AM UTC
```

Comment:
Can you add workflow_dispatch here too? I'd like a way to trigger non-default jobs on-demand ("try jobs"), and that triggering could be part of that.